api/src/main/java/io/strimzi/api/kafka/model/KafkaResources.java

```diff
@@ -0,0 +1,187 @@
+        return clusterName + "-cluster-ca";
+    }
+
+    /**
+     * Returns the name of the Clients CA certificate {@code Secret} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Clients CA certificate {@code Secret}.
+     */
+    public static String clientsCaCertificateSecretName(String clusterName) {
+        return clusterName + "-clients-ca-cert";
+    }
+
+    /**
+     * Returns the name of the Clients CA key {@code Secret} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding Clients CA key {@code Secret}.
+     */
+    public static String clientsCaKeySecretName(String clusterName) {
+        return clusterName + "-clients-ca";
+    }
+
+    /**
+     * Returns the name of the internal bootstrap {@code Service} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding bootstrap {@code Service}.
+     */
+    public static String internalBootstrapServiceName(String clusterName) {
+        return clusterName + "-kafka-bootstrap";
+    }
+
+    /**
+     * Returns the socket address (<em>&lt;host&gt;</em>:<em>&lt;port&gt;</em>)
+     * of the internal plain bootstrap {@code Service} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The socket address of the corresponding bootstrap {@code Service}.
+     * @see #internalTlsBootstrapConnection(String)
+     */
+    public static String internalPlainBootstrapConnection(String clusterName) {
+        return internalBootstrapServiceName(clusterName) + ":9092";
+    }
+
+    /**
+     * Returns the socket address (<em>&lt;host&gt;</em>:<em>&lt;port&gt;</em>)
+     * of the internal TLS bootstrap {@code Service} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The socket address of the corresponding bootstrap {@code Service}.
+     * @see #internalPlainBootstrapConnection(String)
+     */
+    public static String internalTlsBootstrapConnection(String clusterName) {
+        return internalBootstrapServiceName(clusterName) + ":9093";
+    }
+
+    /**
+     * Returns the name of the (headless) brokers {@code Service} for a {@code Kafka} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code Kafka} resource.
+     * @return The name of the corresponding brokers {@code Service}.
+     */
+    public static String brokersServiceName(String clusterName) {
+        return clusterName + "-kafka-brokers";
+    }
+
+    public static String brokerConnection(String clusterName) {
```

Comment:
I do not think anyone should be using this. You should remove this method unless you need it somewhere internally.